src/components/OpenCreateForm.jsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
```

Comment:
Actually we can create a general OpenModalComponent and pass just {children} instead of <Form />